### PR TITLE
#4382: Documenting rsync exclude-paths in site aliases

### PR DESCRIPTION
**Pull request by jcmartinez**
Only adds a snippet of code to an example file.

### CODE REVIEW

**Comment by weitzman** [comment]
I'm not sure what this adds beyond the sql:sync example above it. Clearly we cant add examples for all commands.

**Comment by jcmartinez** [comment]
Yes, you're right, it only adds a sql:sync example with the use of excluded paths.
I'd like to point out that following the existing example, it was impossible for me to get it right without coming here and asking. Sometimes, because we know the way that something works, it seems obvious to us, but not necessarily to everybody.
I'm of the opinion that it doesn't hurt to have open source projects abundantly documented.

**Comment by weitzman** [comment]
Right. How can we expand the sql:sync example so it might have pointed you
in the right direction? We can add comments in the yml.

**Comment by greg-1-anderson** [comment]
I think the thing about rsync that's confusing is that you can't add arguments to aliases, only options. Most rsync "options" are considered arguments to Drush (as they appear after the `--`), but `exclude-paths` is different, as it is processed directly by Drush, and ergo is an option, and ergo can be included in an alias. Same with `mode`.

sql:sync, on the other hand, is a native Drush command, and does not have this issue.

I am +1 on including this example with a comment on passthru options (arguments as options), because rsync is unusual in this respect. Feel free to close again if you still feel it is superfluous, though.

**Comment by weitzman** [comment]
OK, lets add that comment about passthru options and merge,

**Comment by greg-1-anderson** [comment]
Will do.

**Comment by greg-1-anderson** [comment]
My explanation was a bit wordy, but there it is.
